packages/config/config/devices/0x0441/ibt4zwave.json

```diff
@@ -0,0 +1,289 @@
+// NICE Spa IBT4ZWAVE
+// BusT4-Z-Wave interface
+{
+	"manufacturer": "NICE Spa",
+	"manufacturerId": "0x0441",
+	"label": "IBT4ZWAVE",
+	"description": "BusT4-Z-Wave interface",
+	"devices": [
+		{
+			"productType": "0x2400",
+			"productId": "0x1000",
+			"zwaveAllianceId": 3837
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"30[0xff]": {
+			"label": "Alarm Configuration - 1st Slot Notification Type",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 22,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+								{
+					"label": "Smoke Alarm",
+					"value": 1
+				},
+				{
+					"label": "CO Alarm",
+					"value": 2
+				},
+				{
+					"label": "Heat Alarm",
+					"value": 4
+				},
+				{
+					"label": "Water Alarm",
+					"value": 5
+				},
+				{
+					"label": "Access Control",
+					"value": 6
+				},
+				{
+					"label": "Home Security",
+					"value": 7
+				},
+				{
+					"label": "Power Management",
+					"value": 8
+				},
+				{
+					"label": "System",
+					"value": 9
+				},
+				{
+					"label": "Emergency Alarm",
+					"value": 10
+				},
+				{
+					"label": "Clock",
+					"value": 11
+				},
+				{
+					"label": "Appliance",
+					"value": 12
+				},
+				{
+					"label": "Home Health",
+					"value": 13
+				},
+				{
+					"label": "Siren",
+					"value": 14
+				},
+				{
+					"label": "Water Valve",
+					"value": 15
+				},
+				{
+					"label": "Weather Alarm",
+					"value": 16
+				},
+				{
+					"label": "Irrigation",
+					"value": 17
+				},
+				{
+					"label": "Gas Alarm",
+					"value": 18
+				},
+				{
+					"label": "Pest Control",
+					"value": 19
+				},
+				{
+					"label": "Light Sensor",
+					"value": 20
+				},
+				{
+					"label": "Water Quality Monitoring",
+					"value": 21
+				},
+				{
+					"label": "Home Monitoring",
+					"value": 22
+				}
+			]
+		},
+    "30[0xff00]": {
+			"label": "Alarm Configuration - 1st Slot Notification Status",
```

Comment:
Did you see https://github.com/zwave-js/node-zwave-js/pull/2525#issuecomment-832471286 ?
I _think_ this _should_ be the Notification Event (with 255 meaning "Any Event", whatever that does 🤷🏻‍♂️)
At least if we trust the explanation in the manual.